src/esp/bindings/SimBindings.cpp

```diff
@@ -93,10 +93,14 @@ void initSimBindings(py::module& m) {
       .def_property_readonly(
           "gfx_replay_manager", &Simulator::getGfxReplayManager,
           R"(Use gfx_replay_manager for replay recording and playback.)")
+      .def_property_read_only("sensor_suite", &Simulator::getSensorSuite)
```

Comment:
This is going to conflict with the Simulator python class. Furthermore, the Python sensorsuite is entirely seperate from the C++ one, we need to do more work before merging them.

Reply:
Thanks for the feedback Aaron, I see now that the python sensor suite is a Dict[str, hsim.Sensor], while the C++ SensorSuite is std::map<std::string, Sensor::ptr>. In this PR, we are making changes that we'd like to see on the C++ side, but I don't know enough about the python side to know if we'd like to see the same changes. It seems like the functionality to just add sensors already exists, right? Do you think I should modify how python sensors work to add them using this SensorFactory method, or just leave it as is?

Reply:
Oh I think I understand now, we should implement a similar SensorFactory method on the python side to create sensors.